components/Tabs/Tabs.js

```diff
@@ -1,38 +1,42 @@
-
 class TabItem {
   constructor(element) {
-    // attach dom element to object. Example in Tabs class
+    this.element = element;// attach dom element to object. Example in Tabs class
+    this.classList = element.classList;
   }
 
   select() {
-    // should use classList
+    this.classList.add("Tabs__item-selected");// should use classList
   }
 
   deselect() {
-    // should use classList
+    this.classList.remove("Tabs__item-selected");// should use classList
   }
 }
 
 class TabLink {
   constructor(element, parent) {
-    this.element;// attach dom element to object
-    this.tabs;// attach parent to object
-    this.tabItem;// assign this to the associated tab using the parent's "getTab" method by passing it the correct data
+    this.element = element;// attach dom element to object
+    this.tabs = parent;// attach parent to object
+    
+    this.tabItem = this.tabs.getTab(this.element.dataset.tab);
+    // assign this to the associated tab using the parent's "getTab" method by passing it the correct data
```

Comment:
It's a good idea to be consistent with comments, you have some to the side other below. Keep all of them the same.